### PR TITLE
Added documentation on how to add private CA for Helm Chart Repositories in Rancher v2.5.x / v2.6.x

### LINKED ISSUE #3740: [Rancher2] Add documentation on how to add private CA for Helm Chart Repositories in Rancher v2.5.x / v2.6.x
**Request Summary:**
The [Rancher v2.5.x](https://rancher.com/docs/rancher/v2.5/en/helm-charts/) and [v2.6.x](https://rancher.com/docs/rancher/v2.6/en/helm-charts/) documentation on Helm Chart Repositories do not mention the topic of adding a private CA, where the Helm Chart Repo endpoint uses a certificate signed by a private CA.

**Details:**
For HTTP base chart repositories the user needs to add a base64 encoded copy of the CA certificate in DER format (i.e. `openssl x509 -outform der -in ca.pem | base64 -w0`) to the spec.caBundle field of the Chart Repository. You can add this by clicking Edit YAML for the Chart Repository, and setting per the following example:
```
[...]
spec:
  caBundle: MIIFXzCCA0egAwIBAgIUWNy8WrvSkgNzV0zdWRP79j9cVcEwDQYJKoZIhvcNAQELBQAwPzELMAkGA1UEBhMCVVMxCzAJBgNVBAgMAkNBMRQwEgYDVQQKDAtNeU9yZywgSW5jLjENMAsGA1UEAwwEcm9vdDAeFw0yMTEyMTQwODMyMTdaFw0yNDEwMDMwODMyMTdaMD8xCzAJBgNVBAYTAlVTMQswCQYDVQQIDAJDQTEUMBIGA1UECgwLTXlPcmcsIEluYy4xDTALBgNVBAMMBHJvb3QwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQDPUF2+8qIdwTTGgb3RDW/bKrTB2J+mj9ZTOpjt2801CWWb2vSJQZTh2vZuvabzD/Z/vwA0gnrb5FpRqPKlyjIRPUGk7ZLmWh6qjL6GA1/7kDlr9QXXh5T58+kb+xW7cc9HuI2WGUhyGI9+5biw/C9dSwwxpqg35xS94mm8113kODYHNK6DsxkYmg7IX4hpovN78MTkZSWbSYi7EMJ4wEykS/37PK973zKZXA0EHjwHzK5D6DZ5+WqhWMyHIVgfLb/PkdReiIRbKuypKH8TxDIwgdttr4asiHdI+sk5BgOqpALJtHBJ9rpEcRs2H1nXpmKKVwCWgbU2JdSbuX9qKH+Chk5ei0ANWnM9kN4wDD843Zo9Do5gdB5FJIRfhlMtu7TRP6XM6dpoZ1DbDvpUhCoCC1mtyZxG0iRiXkCBlPkf+OjuoBqLynaND2iKodsaYASOiHQW1uPlU48l3ssG/v91C/QGsaAY9XznMQyuqsRWO8F0v1hwKiZ8Uq92ybYx6613mAYP+XZa/3sNJfs87GrSP4792Ko0rCswD8k0kgNcLc9ty3VittvFUdYBlVe+dDp6abE83DAWzrrlhnyp4VPrw/xpsCBCbUub9YShwr154DnUlCCCkOyilCNcbAyFsUCdGmJ+nYo92loj8sV3WXtpqQTGlgBK7/73NtqOGnFQ4QIDAQABo1MwUTAdBgNVHQ4EFgQUEA79+Y5uBwcXnz6FXMjaWGdMd8MwHwYDVR0jBBgwFoAUEA79+Y5uBwcXnz6FXMjaWGdMd8MwDwYDVR0TAQH/BAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAgEAu0kPM2DqKiIJY7hlEU6deRX+XcLcFxeLTonYer6VeAfIpRc5IjzVSqqspCBrz8yARTEdPMzIA04RXzwXRcI/m1PvEQIrFai0IVFic092j2ldzIaL7a33AdRRnb5Ej58c/bv1F4iy/ET+O1gUgiIj8+6aJhoqrbkvaTHxpF/xhnQAI+AqtqzzruNc3KIC6jseqGTGK0SGbHwA1NU7KMl55i2TKtTppjbT3r9T2Js8Yl3yVsJ+o5rksRRusAHLWJGvsPQ+XmW25SE2YO/7RXy+T/tjum1qAP5Db2SEBGQQag8eCK+F6aK2OabGCdLu/+wh6JlQTzpdHibYqk1kiPMJQeShyoCNwAEacoHYp/nhv23REt0az/qX2SLQc6GWzyTv9WBWNl9Rig8bTPpACThu9eJY+W0uRaqgOYJF/qXFu7EEZ9dr4xKjqm/mouxb2kdK2x9IP5L/sUmnxNZ8dAqf7XDwRKKM77e9JvQmvrB060HSvnDxZ/tNXt/WPJr/PgEB3hQdInDWYMg7vGO0Oz00G5kWg0sJ0ZTSoA10ZwdjIdGEeKlj1NlPyAqpQ+uDnmx6DW+zqfYtLnc/g6GuLLVPamraqN+gyU8CHwAWPNjZonFN9Vpg0PIk1I2zuOc4EHifoTAXSpnjfzfyAxCaZsnTptimlPFJJqAMj+FfDArGmr4=
[...]
```
For git-based chart repositories it is not currently possible to add a private CA, per github.com/rancher/rancher/issues/31676 and SURE-3086. For git based chart repositories with a certificate signed by a private CA, a user must currently disable TLS verification:
```
[...]
spec:
  insecureSkipTLSVerify: true
[...]
```

### PR DESCRIPTION
**Pull request by jtravee**
Closes #3740

### CODE REVIEW

File: content/rancher/v2.5/en/helm-charts/_index.md [hunk 1]
```diff
@@ -50,6 +50,20 @@ From the left sidebar select _"Repositories"_.
...
 These items represent helm repositories, and can be either traditional helm endpoints which have an index.yaml, or git repositories which will be cloned and can point to a specific branch. In order to use custom charts, simply add your repository here and they will become available in the Charts tab under the name of the repository.
 
+To add a private CA for Helm Chart repositories:
+
+- **HTTP-based chart repositories**: You must add a base64 encoded copy of the CA certificate in DER format to the spec.caBundle field of the chart repo, such as `openssl x509 -outform der -in ca.pem | base64 -w0`. Click `Edit YAML` for the chart repo and set, as in the following example:</br>
+    ```
+    [...]
+    spec:
+        caBundle:
+    MIIFXzCCA0egAwIBAgIUWNy8WrvSkgNzV0zdWRP79j9cVcEwDQYJKoZIhvcNAQELBQAwPzELMAkGA1UEBhMCVVMxCzAJBgNVBAgMAkNBMRQwEgYDVQQKDAtNeU9yZywgSW5jLjENMAsGA1UEAwwEcm9vdDAeFw0yMTEyMTQwODMyMTdaFw0yNDEwMDMwODMyMT
```
Comment (btat) [inline content/rancher/v2.5/en/helm-charts/_index.md:60]: There are some extra whitespaces at the start of this line.

File: content/rancher/v2.5/en/helm-charts/_index.md [hunk 2]
```diff
@@ -50,6 +50,20 @@ From the left sidebar select _"Repositories"_.
...
+    ...
+    nDxZ/tNXt/WPJr/PgEB3hQdInDWYMg7vGO0Oz00G5kWg0sJ0ZTSoA10ZwdjIdGEeKlj1NlPyAqpQ+uDnmx6DW+zqfYtLnc/g6GuLLVPamraqN+gyU8CHwAWPNjZonFN9Vpg0PIk1I2zuOc4EHifoTAXSpnjfzfyAxCaZsnTptimlPFJJqAMj+FfDArGmr4=
```
Comment (btat) [inline content/rancher/v2.5/en/helm-charts/_index.md:62]: There are some extra whitespaces at the start of this line.

File: content/rancher/v2.5/en/helm-charts/_index.md [hunk 3]
```diff
@@ -50,6 +50,20 @@ From the left sidebar select _"Repositories"_.
...
+    [...]
+    ```
+
+- **Git-based chart repositories**: It is not currently possible to add a private CA. For git-based chart repositories with a certificate signed by a private CA, you must disable TLS verification. Click `Edit YAML` for the chart repo, and add the key/value pair: `spec.insecureSkipTLSVerify: true`.
```
Comment (btat) [inline content/rancher/v2.5/en/helm-charts/_index.md:66]: Could this be converted to a codeblock style example like above?

File: content/rancher/v2.5/en/helm-charts/_index.md [hunk 4]
```diff
@@ -50,6 +50,20 @@ From the left sidebar select _"Repositories"_.
 
 These items represent helm repositories, and can be either traditional helm endpoints which have an index.yaml, or git repositories which will be cloned and can point to a specific branch. In order to use custom charts, simply add your repository here and they will become available in the Charts tab under the name of the repository.
 
+To add a private CA for Helm Chart repositories:
+
+- **HTTP-based chart repositories**: You must add a base64 encoded copy of the CA certificate in DER format to the spec.caBundle field of the chart repo, such as `openssl x509 -outform der -in ca.pem | base64 -w0`. Click `Edit YAML` for the chart repo and set, as in the following example:</br>
```
Comment (btat) [inline content/rancher/v2.5/en/helm-charts/_index.md:55]:
```suggestion
- **HTTP-based chart repositories**: You must add a base64 encoded copy of the CA certificate in DER format to the spec.caBundle field of the chart repo, such as `openssl x509 -outform der -in ca.pem | base64 -w0`. Click **Edit YAML** for the chart repo and set, as in the following example:</br>
```

File: content/rancher/v2.5/en/helm-charts/_index.md (hunk 3 above)
Comment (btat) [inline content/rancher/v2.5/en/helm-charts/_index.md:66]:
```suggestion
- **Git-based chart repositories**: It is not currently possible to add a private CA. For git-based chart repositories with a certificate signed by a private CA, you must disable TLS verification. Click **Edit YAML** for the chart repo, and add the key/value pair: `spec.insecureSkipTLSVerify: true`.
```

File: content/rancher/v2.6/en/helm-charts/_index.md [hunk 5]
```diff
@@ -25,6 +25,20 @@ From the left sidebar select _"Repositories"_.
 
 These items represent helm repositories, and can be either traditional helm endpoints which have an index.yaml, or git repositories which will be cloned and can point to a specific branch. In order to use custom charts, simply add your repository here and they will become available in the Charts tab under the name of the repository.
 
+To add a private CA for Helm Chart repositories:
+
+- **HTTP-based chart repositories**: You must add a base64 encoded copy of the CA certificate in DER format to the spec.caBundle field of the chart repo, such as `openssl x509 -outform der -in ca.pem | base64 -w0`. Click `Edit YAML` for the chart repo and set, as in the following example:</br>
```
Comment (btat) [inline content/rancher/v2.6/en/helm-charts/_index.md:30]:
```suggestion
- **HTTP-based chart repositories**: You must add a base64 encoded copy of the CA certificate in DER format to the spec.caBundle field of the chart repo, such as `openssl x509 -outform der -in ca.pem | base64 -w0`. Click **Edit YAML** for the chart repo and set, as in the following example:</br>
```

File: content/rancher/v2.6/en/helm-charts/_index.md [hunk 6]
```diff
@@ -25,6 +25,20 @@ From the left sidebar select _"Repositories"_.
...
+    ```
+    [...]
+    spec:
+        caBundle:
+    MIIFXzCCA0egAwIBAgIUWNy8WrvSkgNzV0zdWRP79j9cVcEwDQYJKoZIhvcNAQELBQAwPzELMAkGA1UEBhMCVVMxCzAJBgNVBAgMAkNBMRQwEgYDVQQKDAtNeU9yZywgSW5jLjENMAsGA1UEAwwEcm9vdDAeFw0yMTEyMTQwODMyMTdaFw0yNDEwMDMwODMyMT
```
Comment (btat) [inline content/rancher/v2.6/en/helm-charts/_index.md:35]: There are some extra whitespaces at the start of this line.

File: content/rancher/v2.6/en/helm-charts/_index.md [hunk 7]
```diff
@@ -25,6 +25,20 @@ From the left sidebar select _"Repositories"_.
...
+    ...
+    nDxZ/tNXt/WPJr/PgEB3hQdInDWYMg7vGO0Oz00G5kWg0sJ0ZTSoA10ZwdjIdGEeKlj1NlPyAqpQ+uDnmx6DW+zqfYtLnc/g6GuLLVPamraqN+gyU8CHwAWPNjZonFN9Vpg0PIk1I2zuOc4EHifoTAXSpnjfzfyAxCaZsnTptimlPFJJqAMj+FfDArGmr4=
```
Comment (btat) [inline content/rancher/v2.6/en/helm-charts/_index.md:37]: There are some extra whitespaces at the start of this line.

File: content/rancher/v2.6/en/helm-charts/_index.md [hunk 8]
```diff
@@ -25,6 +25,20 @@ From the left sidebar select _"Repositories"_.
...
+    [...]
+    ```
+
+- **Git-based chart repositories**: It is not currently possible to add a private CA. For git-based chart repositories with a certificate signed by a private CA, you must disable TLS verification. Click `Edit YAML` for the chart repo, and add the key/value pair: `spec.insecureSkipTLSVerify: true`.
```
Comment (btat) [inline content/rancher/v2.6/en/helm-charts/_index.md:41]:
```suggestion
- **Git-based chart repositories**: It is not currently possible to add a private CA. For git-based chart repositories with a certificate signed by a private CA, you must disable TLS verification. Click **Edit YAML** for the chart repo, and add the key/value pair: `spec.insecureSkipTLSVerify: true`.
```

**Review by btat — CHANGES_REQUESTED** [review]

File: content/rancher/v2.5/en/helm-charts/_index.md (hunk 1 above)
Comment (btat) [reply content/rancher/v2.5/en/helm-charts/_index.md:60]: Might be a Github rendering issue

File: content/rancher/v2.5/en/helm-charts/_index.md (hunk 2 above)
Comment (btat) [reply content/rancher/v2.5/en/helm-charts/_index.md:62]: Might be a Github rendering issue

File: content/rancher/v2.6/en/helm-charts/_index.md (hunk 6 above)
Comment (btat) [reply content/rancher/v2.6/en/helm-charts/_index.md:35]: Might be a Github rendering issue

File: content/rancher/v2.6/en/helm-charts/_index.md (hunk 7 above)
Comment (btat) [reply content/rancher/v2.6/en/helm-charts/_index.md:37]: Might be a Github rendering issue

File: content/rancher/v2.5/en/helm-charts/_index.md (hunk 3 above)
Comment (jtravee) [reply content/rancher/v2.5/en/helm-charts/_index.md]: Revised as suggested.

File: content/rancher/v2.5/en/helm-charts/_index.md (hunk 4 above)
Comment (jtravee) [reply content/rancher/v2.5/en/helm-charts/_index.md]: Revised as suggested.

File: content/rancher/v2.6/en/helm-charts/_index.md (hunk 5 above)
Comment (jtravee) [reply content/rancher/v2.6/en/helm-charts/_index.md]: Revised as suggested.

File: content/rancher/v2.6/en/helm-charts/_index.md (hunk 8 above)
Comment (jtravee) [reply content/rancher/v2.6/en/helm-charts/_index.md]: Revised as suggested.

File: content/rancher/v2.5/en/helm-charts/_index.md (hunk 3 above)
Comment (jtravee) [reply content/rancher/v2.5/en/helm-charts/_index.md]: Revised as suggested.

**Review by btat — APPROVED** [review]

**Review by jiaqiluo — APPROVED** [review]

**Pull request merged**